cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -767,6 +769,113 @@ const char* gdv_fn_initcap_utf8(int64_t context, const char* data, int32_t data_
 }
 }
 
+GANDIVA_EXPORT
+const char* gdv_fn_from_unixtime_int64(int64_t context, gdv_timestamp in,
+                                       int32_t* out_len) {
+  const char* pattern = "yyyy-MM-dd hh:mm:ss";
+  const int length = strlen(pattern);
```

Comment:
I think you should insert the pattern length manually and change the name to `pattern_len`.

I think magic numbers are bad when they are inserted without context, but in that case you know that number refers to the pattern above. It would avoid a call to the `strlen` function too.

Reply:
It make senses, I will apply your suggestion.